### ISSUE TITLE
Don't start on OnePlus 5T

### ISSUE THREAD

**Issue by BKNadir** [SachinVin/citra_android#198]
Hello there. I've tried to run the app but it doesn't start on my phone. Just getting an error of the run with a popup message. I've a OnePlus 5T 128go smartphone on android 9.0. I've never launch the app with success.

**Comment by mushinako** [comment]
Please elaborate. What error and what message? Working OK-ish on my 5T on OB 24, although the buttons are out of place.

**Comment by BKNadir** [comment]
Yep, The app doesnt start at all. It just returns to home screen. Sometime an alert pop on screen. The alert said that the app has crashd

**Comment by Nishant765** [comment]
Can you tell what games crashed?
